crates/pathfinder/Cargo.toml

```diff
@@ -41,6 +42,7 @@ reqwest = { version = "0.11.13", features = ["json"] }
 rusqlite = { version = "0.28.0", features = ["bundled"] }
 semver = "1.0.14"
 serde = { version = "1.0.149", features = ["derive"] }
+serde_json = { version = "1.0.89", features = ["arbitrary_precision", "raw_value"] }
```

Comment:
Was this always misssing from `[deps]` or why was this now required?

Reply:
It's a new dependency -- we have to do some JSON manipulation on the output of the Sierra -> CASM compiler because the output of the compiler is _not_ exactly the format that's used on Starknet...